### ISSUE TITLE
[Feature] Per-game Steam Input override for non-Steam games

### ISSUE THREAD

**Issue by DanMan** [ValveSoftware/steam-for-linux#8679]
Like they exists for Steam games.

I've had this issue with an emulator which can't find my generic gamepad, if launched from Steam with Steam Input enabled for generic gamepads. Solution was to globally disable Steam Input for those gamepads. But I need it enabled for games on Steam, so I currently have to switch it back and forth depending on what I want to play.

So if you added the tab from the game properties where you can disable Steam Input per-game, that'd be very helpful.

P.S.: Strangely enough the gamepad works either way, if the non-Steam game entry points to Lutris instead which then goes on to launch the game in the emulator.

I'm using the RPM version of Steam on Fedora 35.

**Comment by cobalt2727** [comment]
Having the same issue on endeavourOS, with the version of RetroArch installed from `pacman`. This feature would be greatly appreciated.

tl;dr RetroArch works perfectly with my Steam Controller when Steam itself isn't running, but any time Steam is open in the background then the Steam Controller acts as both a gamepad for RetroArch and uses my desktop mouse configuration at the same time.
